test/synth_test_list.go

```diff
@@ -218,24 +238,25 @@ func synthACLTestsList() []testCase {
 		{
 			testName: "acl_vpe_tf",
 			args: &command{
-				cmd:        synth,
+				cmd:        synthesis,
 				subcmd:     acl,
 				config:     sgTesting3Config,
 				spec:       aclVpeSpec,
 				outputFile: "%s/acl_vpe_tf/nacl_expected.tf",
 			},
+			blockedWarning: utils.Ptr("\n"),
```

Comment:
where does this new line come from?

Reply:
`cmd.Println()`. changed to `cmd.Print()`.